src/dbspro/cli/run.py

```diff
@@ -0,0 +1,88 @@
+"""
+Run DPS-Pro pipeline
+"""
+
+import logging
+import os
+from pathlib import Path
+import sys
+import pkg_resources
+from snakemake import snakemake
+
+from dbspro.utils import available_cpu_count
```

Comment:
I guess you are referencing the COLLOSAL available_cpu_count method i added. Yes that would probably be enough but my thinking was as this was something that we could hide in utils the size did not really matter. 

Also this was build to work on whatever system it was deployed on, maybe that is overkill to include, but I am thinking that it could not hurt. But if you think its too much we could reduce this to your suggestion.